### PR TITLE
Add meetups search

### PR DESCRIPTION
**Pull request by kgabryje**

### CODE REVIEW

File: landing-pages/src/js/meetupsList.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+  const templateElement = document.createElement("div");
+  templateElement.innerHTML = templateText;
+
+  const listItems = root.querySelector(".list-items");
+  const searchBox = root.querySelector("[type='search']");
+  const moreButton = root.querySelector(".more");
+
+  let currentPage = 1;
+  let currentQuery = "";
+  fetch("/data/meetups.json")
```
Comment (mik-laj) [inline landing-pages/src/js/meetupsList.js]: For me, this file does not exist in the static directory, so it cannot be accessed from the browser.

Comment (kgabryje) [reply landing-pages/src/js/meetupsList.js]: I created it manually (/dist/data/meetups) for testing purposes

Comment (mik-laj) [reply landing-pages/src/js/meetupsList.js]: Can you share the latest version of this change?

Comment (kgabryje) [reply landing-pages/src/js/meetupsList.js]: It's already the latest version (https://github.com/apache/airflow-site/pull/103/commits/6bc9f3cdca3af4914571496e1e0d3de69302c8b8)

**Pull request merged**